cmd/notary/tuf.go

```diff
@@ -268,7 +275,36 @@ func (t *tufCommander) tufInit(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	rootKeyList := nRepo.CryptoService.ListKeys(data.CanonicalRootRole)
+	var rootKeyList []string
+
+	if t.rootKey != "" {
```

Comment:
This addition is pushing this function right up to the cyclomatic complexity threshold before goreportcard marks it as a problem. The interior of this `if` blocks seems ideally suited to be broken out as its own function.